hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoTransferHandler.java

```diff
@@ -616,10 +628,110 @@ public Fees calculateFees(@NonNull final FeeContext feeContext) {
         return feeContext
                 .feeCalculator(subType)
                 .addBytesPerTransaction(bpt)
+                .addRamByteSeconds(associationsRbs)
```

Comment:
Before, the logic for paying association fees was in CryptoCreateHandler the price per association slot, was about 0.0018 when this was paid during a CryptoCreate.

Now when moving it here and adding it to RamByteSeconds, it could introduce changes in the price per slot, as RamBytesSeconds are cheaper for transfers, as defined in the feeSchedules.

We estimated that instead of paying 0.0018, a slot could cost the users about 0.00005.
Is this change okay? Or should we think of some MULTIPLIER to get closer to the price per slot when it was paid during CryptoCreate.

What are your opinions @Neeharika-Sompalli @tinker-michaelj